base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1037,19 +1036,25 @@ public X509CertImpl getCACert() throws EBaseException {
             return caCertImpl;
         }
 
-        String cert = mConfig.getString("signing.cert");
-        logger.debug("CertificateAuthority: CA signing cert: " + cert);
+        String certnickname = mConfig.getString("signing.certnickname");
+        String tokennickname = mConfig.getString("signing.tokenname");
 
-        if (StringUtils.isEmpty(cert)) {
+        String certName = tokennickname + ":" + certnickname;
```

Comment:
I think usually we only use the token name if it's not an internal token, so something like this:
```
String certName;
if (CryptoUtil.isInternalToken(tokenname)) {
    certName = certnickname;
} else {
    certName = tokennickname + ":" + certnickname;
}
```